src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -310,9 +327,12 @@ pub async fn merge_imms_in_memory(
         inner: Arc::new(SharedBufferBatchInner::new_with_multi_epoch_batches(
             epochs,
             merged_payload,
+            smallest_table_key,
+            largest_table_key,
             kv_count,
             merged_imm_ids,
             range_tombstone_list,
+            collector.get_all_tombstones(),
```

Comment:
Do we still need to keep origin `range_tombstone_list` ?

Reply:
> Do we still need to keep origin `range_tombstone_list` ?

fixed in #9255